gitlab/__init__.py

```diff
@@ -568,7 +568,7 @@ def update(self, obj, **kwargs):
         params.update(kwargs)
         missing = []
         for k in itertools.chain(obj.requiredUrlAttrs,
-                                 obj.requiredCreateAttrs):
+                                 obj.requiredUpdateAttrs):
```

Comment:
On most objects update and create attributes are the same, so I'd prefer to keep the possibility to use requiredCreateAttrs if requiredUpdateAttrs is not defined.

Your change is at the right place, but you need to add a test to make sure you use the proper attribute.

I hope this explanation is clear enough.